src/website/app/pages/Home/index.js

```diff
@@ -115,7 +114,7 @@ const gettingStartedTheme = {
   SiteLink_color_hover: siteColors.yellow_hover
 };
 const buttonTheme = {
-  Button_fontWeight: mineralTheme.fontWeight_regular,
+  Button_fontWeight: 500,
```

Comment:
shouldn't this be using the new variable? it seems strange that this file has a bunch of static themes in it where the rest of our stuff uses functions

Reply:
Good points. We should definitely use the new variable. I don't know enough about the theming to identify what needs to be static here, but @kylegach if you want to help me identify what can be improved, I'm happy to do that.